docs/modules/ROOT/pages/how-tos/configuring/build-with-args.adoc

```diff
@@ -0,0 +1,42 @@
+= Passing build arguments
```

Comment:
You need to add this page into the closest `_nav.adoc` as well so that it shows up in the left navigation bar. i.e. [how-tos/_nav.adoc](https://github.com/konflux-ci/docs/blob/main/docs/modules/ROOT/pages/how-tos/_nav.adoc?plain=1).

Reply:
Updated.